### DISCUSSION TITLE
Provide CLI Auto Completion Scripts

### DISCUSSION THREAD

**Discussion by sprankhub** [wardenenv/warden#613] [Ideas]
It would be nice to have auto completion scripts for popular shells like:

- [ ] Bash
- [ ] Fish
- [ ] Zsh

Having a look at https://github.com/netz98/n98-magerun2/tree/develop/res/autocompletion could help with this.

[a comment by an automated account is not shown]

**Comment by sprankhub** [comment]
This is still relevant and I cannot reopen :-/

**Comment by Lunaetic** [comment]
I use oh-my-zsh, and opened https://github.com/davidalger/warden/pull/280 to address zsh autocompletions.
